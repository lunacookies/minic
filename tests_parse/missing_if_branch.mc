func a {
	if (0)

func b {
	if (0) {
		var x = 1;
	} else

func c {}
