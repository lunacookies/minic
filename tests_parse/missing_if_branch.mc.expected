func a {
	if (0)
		<missing>
	
}

func b {
	if (0) {
		var x = 1;
	} else
		<missing>
}

func c {}
tests_parse:16..17: error: missing if statement true branch
tests_parse:16..17: error: missing “}”
tests_parse:57..58: error: missing if statement false branch
tests_parse:57..58: error: missing “}”
